lib/charms/bind/v0/dns_record.py

```diff
@@ -201,7 +200,7 @@ class DNSRecordProviderData(BaseModel):
         dns_entries: list of entries to manage.
     """
 
-    dns_entries: List[DNSProviderData] = Field(min_length=1)
```

Comment:
Out of curiosity, may I ask why the min_length is no longer applicable?

Reply:
The resulting list might be empty if all validations fail